src/garage/torch/algos/maml/maml_ppo.py

```diff
@@ -0,0 +1,85 @@
+"""Model-Agnostic Meta-Learning (MAML) algorithm applied to PPO."""
+import torch
+
+from garage.torch.algos import _Default, PPO
+from garage.torch.algos.maml.maml import MAML
+
+
+class MAMLPPO(MAML):
+    """Model-Agnostic Meta-Learning (MAML) applied to PPO.
+
+    Args:
+        env (garage.envs.GarageEnv): A multi-task environment.
+        policy (garage.torch.policies.base.Policy): Policy.
+        baseline (garage.np.baselines.Baseline): The baseline.
+        inner_lr (float): Adaptation learning rate.
+        lr_clip_range (float): The limit on the likelihood ratio between
+            policies.
+        max_path_length (int): Maximum length of a single rollout.
+        discount (float): Discount.
+        gae_lambda (float): Lambda used for generalized advantage
+            estimation.
+        center_adv (bool): Whether to rescale the advantages
+            so that they have mean 0 and standard deviation 1.
+        positive_adv (bool): Whether to shift the advantages
+            so that they are always positive. When used in
+            conjunction with center_adv the advantages will be
+            standardized before shifting.
+        policy_ent_coeff (float): The coefficient of the policy entropy.
+            Setting it to zero would mean no entropy regularization.
+        use_softplus_entropy (bool): Whether to estimate the softmax
+            distribution of the entropy to prevent the entropy from being
+            negative.
+        stop_entropy_gradient (bool): Whether to stop the entropy gradient.
+        entropy_method (str): A string from: 'max', 'regularized',
+            'no_entropy'. The type of entropy method to use. 'max' adds the
+            dense entropy to the reward for each time step. 'regularized' adds
+            the mean entropy to the surrogate objective. See
+            https://arxiv.org/abs/1805.00909 for more details.
+        meta_batch_size (int): Number of tasks sampled per batch.
+        num_grad_updates (int): Number of adaptation gradient steps.
+
+    """
+
+    def __init__(self,
+                 env,
+                 policy,
+                 baseline,
+                 inner_lr=_Default(3e-4),
+                 lr_clip_range=2e-1,
+                 max_path_length=500,
```

Comment:
this is quite long. most MuJoCo environments use something like 100. is this what you found in ProMP?

Reply:
No. This is the default number of `torch/PPO.py` in garage.

Reply:
i suggest you set it to match typical values from ProMP